### DISCUSSION TITLE
Unable to create an instance of FreeSql.FreeSqlBuilder() in some C# .NET Framework 4.0 project

### DISCUSSION THREAD

**Discussion by HwaChuang** [dotnetcore/FreeSql#1741] [Q&A]
I am able to create an instance of FreeSql.FreeSqlBuilder() in a simple C# .NET Framework 4,0 console project.

However, it crashes when I use it in a larger C# .NET Framework 4.0 project.

Had try to use try-catch to catch exception, but didn't catch any exception when it crashed.

That larger C# project can only be built on remote server in release mode, not my local. So, I am not able to use debugger.

Also, I didn't see the constructor of FreeSqlBuilder in GitHub repo.

Any suggestion for both questions?

**Comment by 2881099** [comment]
UseConnectionString

replace to

UseConnectionFactory
